gui_qt/check_failures_report.py

```
@@ -0,0 +1,583 @@
+    if item.line is not None:
+        location_parts.append(f"第 {item.line} 行")
+    if item.item_id:
+        location_parts.append(f"ID {item.item_id}")
+    location = " / ".join(location_parts) if location_parts else "位置未知"
+
+    lines = [
+        f"- [{safety_level_label(item.status)}] {reason_code_label(item.reason_code)}",
+        f"  {location}",
+    ]
+    if item.error:
+        lines.append(f"  错误：{safe_preview(item.error, max_len=200)}")
```

Ruff (0.15.18)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)